akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -327,38 +336,8 @@ final class ActorMaterializerSettings private (
   val autoFusing:                  Boolean,
   val maxFixedBufferSize:          Int,
   val syncProcessingLimit:         Int,
-  val ioSettings:                  IOSettings) {
-
-  // backwards compatibility when added IOSettings, shouldn't be needed since private, but added to satisfy mima
```

Comment:
Weird, that this was added before. Unfortunately, it wasn't marked private itself, so I wonder if that's a real compatibility issue by now?